src/components/form/select/select.tsx

```diff
@@ -236,26 +257,67 @@ export const Select: React.FC<SelectProps> = ({
           id={id}
           ref={fakeInputRef}
           name={name}
-          defaultValue={selectedOptionValue}
-          hidden={true}
-          aria-label="" // Intentionally left blank
+          multiple={multiple}
+          value={selectValue}
```

Comment:
If there are multiple values, is the resulting (fake) select spec wise equivalant to a native multiple select? What will the value/options look like?